$MirOS: ports/print/cups/patches/patch-cgi-bin_Makefile,v 1.1 2006/06/21 19:25:33 bsiegert Exp $
--- cgi-bin/Makefile.orig	Wed Jul 11 23:46:19 2007
+++ cgi-bin/Makefile	Tue Oct 21 23:34:53 2008
@@ -90,7 +90,7 @@ libcgi.a:	$(LIBOBJS)
 
 admin.cgi:	admin.o ../Makedefs ../cups/$(LIBCUPS) libcgi.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ admin.o libcgi.a $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ admin.o libcgi.a $(LIBS)
 
 
 #
@@ -99,7 +99,7 @@ admin.cgi:	admin.o ../Makedefs ../cups/$
 
 classes.cgi:	classes.o ../Makedefs ../cups/$(LIBCUPS) libcgi.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ classes.o libcgi.a $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ classes.o libcgi.a $(LIBS)
 
 
 #
@@ -108,7 +108,7 @@ classes.cgi:	classes.o ../Makedefs ../cu
 
 help.cgi:	help.o ../Makedefs libcgi.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ help.o libcgi.a $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ help.o libcgi.a $(LIBS)
 
 
 #
@@ -117,7 +117,7 @@ help.cgi:	help.o ../Makedefs libcgi.a
 
 jobs.cgi:	jobs.o ../Makedefs ../cups/$(LIBCUPS) libcgi.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ jobs.o libcgi.a $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ jobs.o libcgi.a $(LIBS)
 
 
 #
@@ -126,7 +126,7 @@ jobs.cgi:	jobs.o ../Makedefs ../cups/$(L
 
 printers.cgi:	printers.o ../Makedefs ../cups/$(LIBCUPS) libcgi.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ printers.o libcgi.a $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ printers.o libcgi.a $(LIBS)
 
 
 #
@@ -135,7 +135,7 @@ printers.cgi:	printers.o ../Makedefs ../
 
 testcgi:	testcgi.o ../Makedefs libcgi.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testcgi.o libcgi.a ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testcgi.o libcgi.a ../cups/libcups.a \
 		$(COMMONLIBS) $(SSLLIBS) $(LIBZ) $(LIBGSSAPI)
 
 
@@ -145,7 +145,7 @@ testcgi:	testcgi.o ../Makedefs libcgi.a 
 
 testhi:	testhi.o ../Makedefs libcgi.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testhi.o libcgi.a ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testhi.o libcgi.a ../cups/libcups.a \
 		$(COMMONLIBS) $(SSLLIBS) $(LIBZ) $(LIBGSSAPI)
 
 
@@ -155,7 +155,7 @@ testhi:	testhi.o ../Makedefs libcgi.a ..
 
 testtemplate:	testtemplate.o ../Makedefs libcgi.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testtemplate.o libcgi.a ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testtemplate.o libcgi.a ../cups/libcups.a \
 		$(COMMONLIBS) $(SSLLIBS) $(LIBZ) $(LIBGSSAPI)
 
 
